### PR TITLE
Add `k8s/pkg/apis` module for third party software

### PR DESCRIPTION
**Pull request by denis-tingajkin**
## Motivation and Context
https://github.com/networkservicemesh/networkservicemesh/issues/1962

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: k8s/go.mod [hunk 1]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/networkservicemesh/networkservicemesh/controlplane v0.2.0
 	github.com/networkservicemesh/networkservicemesh/controlplane/api v0.2.0
 	github.com/networkservicemesh/networkservicemesh/k8s/api v0.2.0
+	github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis v0.0.0
```
Comment (nickolaev) [inline k8s/go.mod]: `v0.0.0` -> `v0.2.0`?

File: test/go.mod [hunk 2]
```diff
@@ -12,6 +12,7 @@ require (
 	github.com/networkservicemesh/networkservicemesh/forwarder v0.2.0
 	github.com/networkservicemesh/networkservicemesh/forwarder/api v0.2.0
 	github.com/networkservicemesh/networkservicemesh/k8s v0.2.0
+	github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis v0.0.0
```
Comment (nickolaev) [inline test/go.mod]: `v0.0.0` -> `v0.2.0`?

Comment (denis-tingajkin) [reply test/go.mod]: thanks, fixed

File: k8s/go.mod (hunk 1 above)
Comment (denis-tingajkin) [reply k8s/go.mod]: thanks, fixed

**Comment by denis-tingajkin** [comment]
@nickolaev 
> shall we already call this v0.3.0?

Probably yes, but I think, we need to do it in separate PR.

**Pull request merged**